server/src/test/java/io/crate/expression/scalar/HasDatabasePrivilegeFunctionTest.java

```diff
@@ -138,12 +138,15 @@ public void test_no_privilege_other_than_connect() {
 
     @Test
     public void test_create_privilege() {
+        assertEvaluate("has_database_privilege('crate', 'crate', 'CREATE')", true);
         assertEvaluate("has_database_privilege('testWithCreate', 'crate', 'CONNECT')", true);
         assertEvaluate("has_database_privilege('testWithCreate', 'crate', 'CREATE')", true);
         assertEvaluate("has_database_privilege('testWithCreate', 'crate', 'CREATE, CONNECT, create')", true);
         assertEvaluate("has_database_privilege('testWithCreate', 'crate', 'CONNECT, temp, CREATE, TEMP')", true);
 
         // Same as above but with session user
+        sqlExpressions = new SqlExpressions(tableSources, null, User.CRATE_USER);
+        assertEvaluate("has_database_privilege('crate', 'crate', 'CREATE')", true);
```

Comment:
> Same as above but with session user

Shouldn't it be then 
`assertEvaluate("has_database_privilege('crate', 'CREATE')", true);`?

Also, unchanged old entries above (with session user TEST_USER_WITH_CREATE) have to be 
`assertEvaluate("has_database_privilege('crate', 'CONNECT')", true);`

to be in sync with the comment?

Reply:
indeed!